python_modules/libraries/dagster-aws/dagster_aws/s3/s3_fake_resource.py

```diff
@@ -36,6 +36,9 @@ def list_objects_v2(self, Bucket, Prefix, *args, **kwargs):
         else:
             return {"KeyCount": 0, "Contents": [], "IsTruncated": False}
 
+    def list_objects(self, Bucket, Prefix, *args, **kwargs):
```

Comment:
we probably want to include a `self.mock_extras.list_objects(*args, **kwargs)` here. by doing that it means that it means that `self.mock_extras.list_objects(*args, **kwargs)` and `self.mock_extras.list_objects_v2(*args, **kwargs)` will both get called by this mock fn. To avoid that, could you factor out the body of `list_objects_v2` into a helper function that both `list_objects_v2` and `list_objects` call after the `mock_extras` call?

Reply:
Ah good point! I missed that, yep updated that, let me know if it looks good